bin/test-workflow/7_app_deploy.sh

```diff
@@ -82,4 +82,7 @@ pushd ../../helm/conjur-app-deploy > /dev/null
 
 popd > /dev/null
 
+announce "LOGS:"
+$cli logs "$(get_pod_name test-app-secretless-broker)" secretless -n "$TEST_APP_NAMESPACE_NAME"
```

Comment:
I dont see the logs in the build maybe they should be added in a different place. Maybe after line 90

Reply:
Should be a -c flag befpre secret;ees